src/context/toast-context.js

```diff
@@ -0,0 +1,26 @@
+import { createContext, useContext, useState } from "react";
+
+const ToastContext = createContext();
+
+const ToastProvider = ({ children }) => {
+  const [toastMsg, setToastMsg] = useState("");
+  const [toastState, setToastState] = useState(false);
+  const [toastBg, setToastBg] = useState("");
```

Comment:
Instead of having 3 toast states, you can have one object which will store all these states as key-value pair. You won't have to pass three states each time.
```js
const [toastVal, setToastVal] = useState({
  msg: '',
  isOpen: false,
  bg: '',
})
 ```

Reply:
Will definitely do that, Thank you for suggestions :)